config/nginx.conf

```diff
@@ -0,0 +1,31 @@
+user  nginx;
+worker_processes  auto;
+
+error_log  /dev/stdout debug;
+pid        /var/run/nginx.pid;
+
+
+events {
+    worker_connections  1024;
+}
+
+
+http {
+    include       /etc/nginx/mime.types;
+    default_type  application/octet-stream;
+
+    log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
+                      '$status $body_bytes_sent "$http_referer" '
+                      '"$http_user_agent" "$http_x_forwarded_for"';
+
+    access_log  /var/log/nginx/access.log  main;
+
+    sendfile        on;
+    #tcp_nopush     on;
+
+    keepalive_timeout  65;
+
+    #gzip  on;
```

Comment:
Same as https://github.com/Intellection/docker-nginx/pull/1/files#r330548357

Reply:
Resolved in https://github.com/Intellection/docker-nginx/pull/1/commits/88290f8275c330515d657e1226a4c179422f7dde.